app/containers/Avatar/index.tsx

```diff
@@ -88,12 +88,7 @@ class AvatarContainer extends React.Component<IAvatar, any> {
 			const observable = record.observe() as Observable<TSubscriptionModel | TUserModel>;
 			this.subscription = observable.subscribe(r => {
 				const { avatarETag } = r;
-				if (this.mounted) {
-					this.setState({ avatarETag });
-				} else {
-					// @ts-ignore
-					this.state.avatarETag = avatarETag;
-				}
+				this.setState({ avatarETag });
```

Comment:
why you remove the `if (this.mounted)` ?

Reply:
It doesn't matter if is mounted or not you're going to update the state for `avatarETag`

Reply:
in this case, should we remove the `this.mounted` ? Since it is using only to this check ?

Reply:
We should, yeah.